src/pages/patientView/mutation/column/CohortColumnFormatter.tsx

```diff
@@ -5,52 +5,65 @@ import DefaultTooltip from 'shared/components/DefaultTooltip';
 import 'rc-tooltip/assets/bootstrap_white.css';
 import {MutationTableRowData} from "../../../../shared/components/mutationTable/IMutationTableProps";
 import {MutSigData} from "../../PatientViewPage";
-
-export interface IVariantCountData {
-    numberOfSamples?:number;
-    geneData?:{ [entrezGeneId:string]: {
-        numberOfSamplesWithMutationInGene?:number,
-        numberOfSamplesWithKeyword?:{ [keyword:string]:number }
-    }};
-};
+import {VariantCountCacheType} from "../../clinicalInformation/CohortVariantCountCache";
 
 export default class CohortColumnFormatter {
 
     public static renderFunction(data:IColumnFormatterData<MutationTableRowData>, columnProps:any) {
```

Comment:
@adufilie @adamabeshouse This might be tricky, since we have different `columnProps` for each column formatter. For now, `columnProps` parameter is mainly used to pass additional data (external data other than core mutation data).

A possible solution to this problem is to extend  `MutationTableRowData` model to contain all the additional data fields. But that would probably require re-architecturing. Currently `MutationTableRowData` is `Array<Mutation>`.